exercises/01.compartmentalisation/README.md

```diff
@@ -112,7 +119,7 @@ Exercise 1: Confidentiality
 ---------------------------
 
 First, we want to protect the secret.
-For the this exercise, we want to move the 
+For the this exercise, we want to move it into a separate comparment from the JavaScript interpreter: we want to minimise the attack surface of the compartment that handles the secret, so we definitely don't want it to share with the potentially vulnerable interpreter.
```

Comment:
I don't think we want to talk about attack surface here.  We are doing this to isolate the secret, not to reduce the attack surface.  We could talk about blast radius, but that's probably too much detail.

Reply:
Better? I just wanted to fix the incomplete sentence.